VideoRental/Customer.java

```diff
@@ -0,0 +1,94 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Customer {
+	private String name;
+
+	private List<Rental> rentals = new ArrayList<Rental>();
```

Comment:
If same name or same rentals exist, it can make problem. Because we can't compare about same data now.
So we must make some PK for check same data.